coderd/users.go

```diff
@@ -194,7 +194,8 @@ func (api *API) postFirstUser(rw http.ResponseWriter, r *http.Request) {
 			Password:        createUser.Password,
 			OrganizationIDs: []uuid.UUID{defaultOrg.ID},
 		},
-		LoginType: database.LoginTypePassword,
+		LoginType:          database.LoginTypePassword,
+		accountCreatorName: "coder",
```

Comment:
@mafredri Are we happy to hardcode this value here?

Reply:
Correct me if I'm wrong, in this case, "coder" indicates the _system_? If so, then 👍 

re hardcoding:

if this is the only place, then I believe it is fine.

Reply:
> Correct me if I'm wrong, in this case, "coder" indicates the system? 

This is correct